community/consistency-check/src/main/java/org/neo4j/consistency/checking/full/CountsBuilderDecorator.java

```diff
@@ -293,9 +293,8 @@ public void check( RelationshipRecord record,
                         }
                     }
                 }
-
-                inner.check( record, engine, records );
             }
+            inner.check( record, engine, records );
```

Comment:
Does this change mean that the `inner.check` is called for every pass? Why is that a good thing?